### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by BlackfyreInc**

### CODE REVIEW

File: hw7/Product.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    public Product(){
+
+    }
+
+    public void pr(){
+        System.out.printf("ID = %d, name = %s, UPC = %d, producer = %s, price = %d, days left = %d", id, name, upc, producer, price, days_left);
+    }
+    public void setType(){
+
```
Comment (mecchmatProjects) [inline hw7/Product.java:26]: what is it? we need setters an getters here...

File: hw7/hw7_2_9.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import java.util.Scanner;
```
Comment (mecchmatProjects) [inline hw7/hw7_2_9.java:1]: looks like this won\t compile

File: hw7/Product.java (hunk 1 above)
Comment (BlackfyreInc) [reply hw7/Product.java:26]: But we do not need setters or getters in this task.

File: hw7/hw7_2_9.java (hunk 2 above)
Comment (BlackfyreInc) [reply hw7/hw7_2_9.java:1]: It's just automatic import of Scanner in Intellij IDEA.

**Pull request closed without merging**